iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -1901,10 +1901,12 @@ public boolean canUseTruncate(org.apache.hadoop.hive.ql.metadata.Table hmsTable,
 
   private boolean hasUndergonePartitionEvolution(Table table) {
     // If it is a table which has undergone partition evolution, return true.
+    // if a table has undergone partition evolution, the current spec is not necessary the latest which can happen
```

Comment:
just keep the part about the current spec, cause it's valid not only for partition evolution case
`the current spec is not necessary the latest which can happen when partition spec was changed to one of table's past specs.`

Reply:
Done